io/src/main/scala/sbt/internal/io/Retry.scala

```diff
@@ -34,19 +34,19 @@ private[sbt] object Retry {
       excludedExceptions: Class[? <: IOException]*,
   ): T = {
     require(limit >= 1, "limit must be 1 or higher: was: " + limit)
-    def filter(e: Exception): Boolean = excludedExceptions match {
+    def filter(e: Throwable): Boolean = excludedExceptions match {
```

Comment:
`excludedExceptions` is still defined as `Class[? <: IOException]`. It should probably be a `Class[? <: Throwable]`.

Reply:
Yea. I didn't want to take chance with bincompat but I guess I can try and see what mima says.